### DISCUSSION TITLE
Fuzzy finder for all current LSP errors/warnings

### DISCUSSION THREAD

**Discussion by lukepighetti** [helix-editor/helix#4181] [Q&A] [answered]
In VSCode there is a pane that shows all LSP errors/warnings. This is helpful for tracking down the effects of a code edit in files that are not currently open. It also makes it easy to navigate to these errors to resolve them.

Would be nice to have a similar feature in helix. Perhaps with fuzzy finding and code preview.

**Comment by Omnikar** [comment, marked as answer]
`<space>g` (file)
`<space>G` (global)

**Reply by lukepighetti** [reply]
Thanks for the quick answer & triage, much appreciated!
